src/amp-story-player.js

```diff
@@ -14,9 +14,11 @@
  * limitations under the License.
  */
 
+import {AmpStoryPlayer} from './amp-story-player-impl';
 import {AmpStoryPlayerManager} from './amp-story-player-manager';
 
-self.onload = () => {
+self.addEventListener('load', () => {
+  customElements.define('amp-story-player', AmpStoryPlayer);
   const manager = new AmpStoryPlayerManager(self);
   manager.loadPlayers();
```

Comment:
This works, but maybe is a little weird?  Shouldn't each custom element manage its own loading when it's connected?

Reply:
Changed it so that each player manages its loading from its `connectedCallback`